tests/devlxd-container

```diff
@@ -114,7 +114,8 @@ kill -9 "${monitorPID}"
 
 # If the guest retrieved the image from the host, the host should emit an "image-retrieved" lifecycle event. The
 # requestor address tells us that this was definitely the guest.
-[ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.address')" = "@devlxd" ]
+# If using newer snap versions, check the protocol instead.
+[ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.protocol')" = "devlxd" ]
```

Comment:
This should work on old and new LXD:
```suggestion
  # Newer LXD versions use the protocol instead of the requestor address to distinguish devlxd requests
  [ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.address')" = "@devlxd" ] || [ "$(grep -wF 'image-retrieved' monitor.json | jq -r '.metadata.requestor.protocol')" = "devlxd" ]
```